src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -52,6 +57,19 @@ export const RequestCodeViewer = ({ indexPattern, json }: RequestCodeViewerProps
     [consoleHref, navigateToUrl]
   );
 
+  const searchProfilerDataUri = compressToEncodedURIComponent(json);
+  const searchProfilerHref = services.share.url.locators
+    .get('SEARCH_PROFILER_LOCATOR')
+    ?.useUrl({ index: indexPattern, loadFrom: `data:text/plain,${searchProfilerDataUri}` });
+  // Check if both the Dev Tools UI and the SearchProfiler UI are enabled.
+  const canShowsearchProfiler =
+    services.application?.capabilities?.dev_tools.show && searchProfilerHref !== undefined;
+  const shouldShowsearchProfilerLink = !!(indexPattern && canShowsearchProfiler);
+  const handlesearchProfilerLinkClick = useCallback(
```

Comment:
nit: typo with capitalisation
```suggestion
  const handleSearchProfilerLinkClick = useCallback(
```